programs/test/selftest.c

```diff
@@ -243,6 +243,9 @@ const selftest_t selftests[] =
 #if defined(MBEDTLS_SHA1_C)
     {"sha1", mbedtls_sha1_self_test},
 #endif
+#if defined(MBEDTLS_SHA224_C)
+    {"sha256", mbedtls_sha224_self_test},
```

Comment:
Hmm, twice is probably not a typo... what reason am I missing from making this `"sha256"` rather than `"sha224"`?

Reply:
Ok it seems that you also spotted the 2nd error I mentioned [above](https://github.com/Mbed-TLS/mbedtls/pull/6784#discussion_r1054540973). This is just a typo due to the fact that I copy-pasted new lines from the already existing ones

Reply:
Ok, typo is what I thought the first time, then the second time I was no longer so sure ;)